templates/opendkim.conf.j2

```diff
@@ -1,24 +1,36 @@
-AutoRestart             Yes
+AutoRestart             yes
 AutoRestartRate         10/1h
-UMask                   002
-Syslog                  yes
-SyslogSuccess           Yes
-LogWhy                  Yes
-ReportAddress           {{ dkim_admin_email }}
-
-Canonicalization        relaxed/simple
-
-ExternalIgnoreList      refile:/etc/opendkim/TrustedHosts
-InternalHosts           refile:/etc/opendkim/TrustedHosts
-KeyTable                refile:/etc/opendkim/KeyTable
-SigningTable            refile:/etc/opendkim/SigningTable
-
+Canonicalization        relaxed/relaxed
+Domain                  *
+ExternalIgnoreList      refile:{{ dkim_opendkim_config_dir }}/TrustedHosts
+InternalHosts           refile:{{ dkim_opendkim_config_dir }}/TrustedHosts
+LogWhy                  yes
 Mode                    sv
```

Comment:
[in Mode](http://www.opendkim.org/opendkim.conf.5.html) `s ` means _sign_ and `v ` means _verifier_. 

This parameter should also be configurable.